docs/content/index.md

```diff
@@ -22,6 +22,14 @@ These environments can be extended, version controlled, and shared, so you can t
     * Storage: 256GB
     * [Colima](https://github.com/abiosoft/colima) or [Docker Desktop](https://www.docker.com/products/docker-desktop/)
 
+    **Next steps:**
+
+    *You’ll need a Docker provider on your system before you can install DDEV.*
+    
+    1. Install Colima (or Docker Desktop) with [recommended settings](users/install/docker-installation.md#macos).
+    1. Install [DDEV for macOS](users/install/ddev-installation.md#macos)
+    1. Launch your [first project](users/project) and start developing 🚀
```

Comment:
I’d number these sequentially, punctuate consistently, and drop the parentheses in item one:

```suggestion
    1. Install Colima or Docker Desktop with [recommended settings](users/install/docker-installation.md#macos).
    2. Install [DDEV for macOS](users/install/ddev-installation.md#macos).
    3. Launch your [first project](users/project) and start developing. 🚀
```